tests/components/ring/test_sensor.py

```diff
@@ -48,3 +52,26 @@ async def test_sensor(hass: HomeAssistant, requests_mock: requests_mock.Mocker)
     )
     assert front_door_wifi_signal_strength_state is not None
     assert front_door_wifi_signal_strength_state.state == "-58"
+
+
+async def test_only_chime_devices(
+    hass: HomeAssistant, requests_mock: requests_mock.Mocker, caplog
+) -> None:
+    """Tests the update service works correctly if only chimes are returned."""
+    requests_mock.get(
+        "https://api.ring.com/clients_api/ring_devices",
+        text=load_fixture("chime_devices.json", "ring"),
+    )
+    await setup_platform(hass, Platform.SENSOR)
+    await hass.async_block_till_done()
+
+    await async_setup_component(hass, "homeassistant", {})
+    await hass.services.async_call(
+        "homeassistant",
+        "update_entity",
+        {ATTR_ENTITY_ID: ["sensor.downstairs_volume"]},
+        blocking=True,
+    )
```

Comment:
Is this to force an update?

We usually skip time using the freezer instead to trigger a "natural" update

Reply:
I've updated the test to trigger a natural update

Reply:
Please use the freezer for this

Reply:
Apologies I thought `async_fire_time_changed` from tests.common was what you meant.  I don't suppose you could point me to a test using the freezer?

Reply:
You need to use both, if you do a global search on freezer you'll find one. (I'm in public transport atm)